azext_iot/central/params.py

```diff
@@ -82,10 +82,16 @@ def load_central_arguments(self, _):
     with self.argument_context("iot central app monitor-events") as context:
         context.argument("timeout", arg_type=event_timeout_type)
         context.argument("properties", arg_type=event_msg_prop_type)
+        context.argument(
```

Comment:
I'd suggest removing `module_id` from the `validate-messages` and `monitor-events` parents, and adding it to the `iot central app` parent instead